src/main/java/uk/gov/companieshouse/chd/order/api/model/MissingImageDeliveriesRequest.java

```diff
@@ -85,6 +87,22 @@ public void setFilingHistoryDate(String filingHistoryDate) {
 		this.filingHistoryDate = filingHistoryDate;
 	}
 
+	public String getFilingHistoryBarcode() {
+		return filingHistoryBarcode;
+	}
+
+	public void setFilingHistoryBarcode(String filingHistoryBarcode) {
+		this.filingHistoryBarcode = filingHistoryBarcode;
+	}
+
+	public String getEntityID() {
+		return entityID;
+	}
+
+	public void setEntityID(String entityID) {
+		this.entityID = entityID;
+	}
+
```

Comment:
Can the formatting on this class be fixed please? Should be spaces instead of tabs